app/assets/javascripts/organization.coffee

```diff
@@ -0,0 +1,19 @@
+window.addAddressRow = (count) ->
```

Comment:
I think figuring out the count belongs in this method, however I think you can potentially drop the drop the need for the count entirely with a few tweaks.